### PR TITLE
Add `view` implementation for `Series`

### LINKED ISSUE #960: Windows Memory Issue for `Series.view`
### Describe the problem

Test Failure during #957. For now, marking as skip for Windows only and will need to fix later. `view` defaults to pandas in this case, but something with the deserialization must be causing an issue, even when `dtype=None`.

### LINKED ISSUE #1295: view function implementation

### PR DESCRIPTION
**Pull request by YarShev**
Signed-off-by: Yaroslav Igoshev <yaroslav.igoshev@intel.com>

## What do these changes do?

- [x] passes `flake8 modin`
- [x] passes `black --check modin`
- [x] signed commit with `git commit -s` 
- [x] Resolves #1295  
- [x] Resolves #960
- [x] tests added and passing

### CODE REVIEW

**Comment by devin-petersohn** [comment]
Also Resolves #960.

File: modin/pandas/test/test_series.py [hunk 1]
```diff
@@ -2778,13 +2778,12 @@ def test_var(data, skipna, ddof):
...
 
 
-@pytest.mark.skipif(sys.platform == "win32", reason="Windows memory issue #960")
-@pytest.mark.parametrize("data", test_data_values, ids=test_data_keys)
-def test_view(data):
-    modin_series, pandas_series = create_test_series(data)
-
-    with pytest.warns(UserWarning):
-        modin_series.view(None)
+def test_view():
```
Comment (devin-petersohn) [inline modin/pandas/test/test_series.py:2781]: It might be good to also test with `float64`.

Comment (YarShev) [reply modin/pandas/test/test_series.py:2781]: Of course, done.

**Comment by YarShev** [comment]
@devin-petersohn , just a friendly reminder.

**Pull request merged**